OTLoginServer/Classes/DatabaseManager.cs

```diff
@@ -58,9 +58,8 @@ public async Task<Account> GetAccount(string email, string password)
 
                     account.Id = dataReader.GetInt32(dataReader.GetOrdinal("id"));
 
-                    int premDays = dataReader.GetInt32(dataReader.GetOrdinal("premdays"));
+                    int premDays = dataReader.GetInt32(dataReader.GetOrdinal("premium_ends_at"));
                     account.IsPremium = premDays > 0;
-                    account.LastLoginTime = dataReader.GetInt64(dataReader.GetOrdinal("lastday"));
```

Comment:
is this not used or why delete?

Reply:
to get the last login time you need to get it from the character table not from the account.